modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -41,8 +41,32 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * GCP: `LoadBalancerService` (with External scope)
 * Bare metal: `NodePortService`
 * Other: `HostNetwork`
++
+[NOTE]
+====
+`HostNetwork` has a `hostNetwork` field with the following default values for the optional binding ports: `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`.
+With the binding ports, you can deploy multiple Ingress Controllers on the same node for the `HostNetwork` strategy.
+
+.Example
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+    kind: IngressController
+    metadata:
+      name: internal
+      namespace: openshift-ingress-operator
+    spec:
+      domain: private.foo.com
```

Comment:
@Amrita42, we're using the `example.com` form for these, not `foo` or any other construction. Thanks!

Reply:
@jboxman-rh this content already exists but I will make that change. Thanks!

Reply:
Thanks @jboxman-rh  good catch!